### DISCUSSION TITLE
I cannot install release 5.3.0.2

### DISCUSSION THREAD

**Discussion by alodeu** [rsjaffe/MIDI2LR#634] [Problems]
Windows says ".. Payload missing "..
![grafik](https://user-images.githubusercontent.com/6894694/154806376-d54dabc7-fdf0-4a37-be2a-de793830f9fd.png)
 Release 5.3.0.1 could install, but has other bugs...

**Comment by rsjaffe** [comment]
Redownload it. I just downloaded it from github and it ran fine.
